.travis.yml

```diff
@@ -11,4 +11,5 @@ before_script:
   - ./.travis_do.sh download_sdk
 
 script:
+  - ./.travis_do.sh test_commits
```

Comment:
should we add a `set -e` somewhere ? [maybe in `before_script` as first line ]
to fail the build ?

by default, travis does not have this enabled for shell scripts, so you could end up with some intermediate failures that can be missed, because the next step succeeds
i saw this on another repo [the intermediate failure not being caught] ;
i don't know for sure if the case is the same here [entirely]